services/web/server/src/simcore_service_webserver/resource_manager/user_sessions.py

```diff
@@ -117,9 +117,9 @@ async def remove_socket_id(self) -> None:
         )
 
         await self._registry.remove_resource(self._resource_key(), _SOCKET_ID_FIELDNAME)
-        await self._registry.set_key_alive(
-            self._resource_key(), _get_service_deletion_timeout(self.app)
-        )
+        # when the tab is closed the alive key is also removed immediately,
```

Comment:
So from my understanding, this issue is somehow connected to session IDs (where there is probably some issue how we work with them cross product/cross tabs) did you test it and are you sure this removal will not cause other side-effect issue? Are you able to reproduce this issue? (its happening so often that I guess it should be reproducable?)

Reply:
You can reproduce this everywhere. Open a project and cose the tab its alive key will still be present in Redis. This makes no sense to me. But I might be wrong.

Reply:
If you are not sure, why are you changing it without verifying? What if this change makes the GC more unstable?